packages/@okta/vuepress-site/docs/guides/social-login/main/line/noemail.md

```diff
@@ -0,0 +1,46 @@
+## Handle users without email addresses
+
```

Comment:
Can we add a blurb about what to do if you can't see a user's email address?

It's my understanding that LINE requires you to do something in order to collect email addresses. See here: https://developers.line.biz/en/docs/line-login/integrate-line-login/#applying-for-email-permission. So my thought is to have something like


> ###

```suggestion
### The user has an email address in LINE, but it's not showing up in Okta
Before you proceed to the next section, test your integration with a user who has an email in LINE. If you don't see their email in Okta, check that you have [email address permission](https://developers.line.biz/en/docs/line-login/integrate-line-login/#applying-for-email-permission) in the channel you are using. If you have confirmed that you have this permission and are still not able to see their emails in Okta, contact support.

### The user doesn't have an email address in LINE
```